inference-engine/src/mkldnn_plugin/nodes/mkldnn_matrix_nms_node.cpp

```diff
@@ -110,7 +110,8 @@ MKLDNNMatrixNmsNode::MKLDNNMatrixNmsNode(const std::shared_ptr<ngraph::Node>& op
     m_postThreshold = attrs.post_threshold;
     m_normalized = attrs.normalized;
     int64_t max_output_boxes_per_class = 0;
-    size_t real_num_classes = m_backgroundClass == -1 ? m_numClasses : m_numClasses - 1;
+    size_t real_num_classes = m_backgroundClass == -1 ? m_numClasses :
```

Comment:
how about change it to "size_t real_num_classes = m_backgroundClass < 0 ? m_numClasses :" ?

Reply:
I think we can use the original code because we have validated the background_class should be [[-1, max)](https://github.com/openvinotoolkit/openvino/blob/7654789451a37f7cab448e6ad7da61976d9355ec/ngraph/core/src/op/matrix_nms.cpp#L44)